### ISSUE TITLE
Apply for Maintainer

### ISSUE THREAD

**Issue by Snailedlt** [devicons/devicon#1471]
### Discussed in https://github.com/devicons/devicon/discussions/1436

<div type='discussions-op-text'>

<sup>Originally posted by **Snailedlt** October  8, 2022</sup>
# Hi everyone! 👋

We are in need of more active maintainers who can take Devicons to the next level 🚀

[Devicons has grown a lot recently](https://star-history.com/#devicons/devicon), and it has grown into an active community of contributors. As such we are in need of more people to review pull requests, label issues and pull requests, update documentation and help guide new contributors.
If you are someone who would like to help out with such things, please go ahead and apply below (one comment per user)!

## ✔️ Maintainer responsibilities

- Review Pull Requests
- Label Issues and Pull Requests
- Close Inactive and/or Duplicate Issues and Pull Requests
- Guide New Contributors

## 📝 How to apply

We suggest you follow the layout\* below:
> \**You can take a look at [my application](https://github.com/devicons/devicon/discussions/1263#discussion-4207441) as a reference.*

**> ABOUT ME**
Tell us about yourself, what are your interests, what skills do you have, and what do you enjoy the most.

**> ME AND DEVICONS**
Let us know why you are a good fit for Devicons, and how you can contribute to Devicons as a maintainer. Have you made any contributions to Devicons yet? If so, tell us about them too, and link to some of the most important ones.

**> WHY I WANT TO JOIN**
Pretty straight forward. Why would you like to join Devicons? Is it to learn, contribute to open-source, add value to your own projects, or something else?

**> MY SOCIALS**
Links to your contact information and/or socials. Pretty much anything that's open and let us contact you in private messages/e-mails.

## 👨‍💻 In which other ways can you contribute?

There are a lot of different stuff to do in Devicons:

- Improve and build out [our website](https://devicon.dev) (`HTML`, `CSS`, `JavaScript`, `Angular`)
- Automate processes (`GitHub Actions`, `CI/CD`, `Python`, `Selenium`, `Gulp.js`)
- Add Icons (`SVG` and `JSON` editing)
- Add more tests to increase the projects robustness and quality (maybe even implement a test framework from scratch)
- Update Our documentation (`GitHub Wiki`, `Markdown`, `HTML`, `CSS`)

**PS:**
We are eligible for [hacktoberfest](https://hacktoberfest.com/), so your maintainer activity and contributions to [Devicons/Devicon](https://github.com/devicons/devicon) through the month of October count towards your hacktoberfest score and rewards.</div>

**Comment by Snailedlt** [comment]
Anyone interested in this?

I'll tag some of the most active contributors in case any of you are interested :)
@weh @BenSouchet @kilian-paquier @lunatic-fox

**Comment by Snailedlt** [comment]
Ofc anyone else are also free to apply and ask questions

**Comment by Snailedlt** [comment]
We're still looking for a few more maintainers btw :)

**Comment by dotarjun** [comment]
Hey @Snailedlt I know I am late but lmk what I need to do to be a maintainer

**Comment by Panquesito7** [comment]
> Hey @Snailedlt I know I am late but lmk what I need to do to be a maintainer

You need to:

- Be active in the community (Discord and GitHub).
- Make plenty of contributions to the codebase or add new icons as well.
- Help the community with their doubts/questions when possible.
- Help in fixing bugs when possible as well.

There's also a lot more you can do, these are just a few things to do. 🙂

**Comment by Snailedlt** [comment]
@dotarjun Same as what @Panquesito7 said :) The only requirement is that you've had meaningful contributions to the community, preferable in the form of reviews and/or code :)

Thanks for the interest!

**Comment by pataquets** [comment]
@Snailedlt Awesome repo. Thanks for sharing all that work.
Just dropping here https:/adoptoposs.org, which might be of interest.
HTH.

**Comment by Snailedlt** [comment]
@pataquets Thanks for the suggestion. I added the project here :D
https://adoptoposs.org/p/c33ba413-e2f8-42ad-8b97-f43517b77ec1

**Comment by Khnaz35** [comment]
Is this repository still being managed?

**Comment by Snailedlt** [comment]
> Is this repository still being managed? 

@Khnaz35 yes, though not as actively as we'd like. Take a look at the develop branch to see the newest additions

**Comment by Yashh56** [comment]
Is the application for maintainer still open?

**Comment by Snailedlt** [comment]
@Yashh56 yes, we are still in need of active maintainers

**Comment by gewaleelek** [comment]
Can I apply for just the permission to help close out request tickets of merged icons? :D
